src/components/FormDropdownSearch/FormDropdownSearch.vue

```diff
@@ -174,7 +181,9 @@ const onFocus = () => {
 
 watch(modelValue, (newValue, oldValue) => {
     if (newValue) {
-        searchValueCached.value = newValue[props.labelKey];
+        searchValueCached.value = props.disableCache
+            ? ''
+            : newValue[props.labelKey];
```

Comment:
Just never set it I guess. Setting any value will trigger at least the proxy wrapped around the value.
```suggestion
        if (!props.disableCache) {
            searchValueCached.value = newValue[props.labelKey];
        }
```